app/Models/Plan/PlanDay.php

```diff
@@ -135,11 +135,10 @@ public function complete(int $user_id = null) : void
         if (is_null($user_id)) {
             $user_id = Auth::user()->id;
         }
-        $completed_item = PlanDayComplete::firstOrNew([
+        PlanDayComplete::updateOrCreate([
```

Comment:
@bradflood, the Laravel documentation recommends using the 'updateOrCreate' method for single records and 'upsert' for multiple records.